script-http-request/README.md

```diff
@@ -0,0 +1,43 @@
+# HTTP Request via Script
+
+Table of Contents
+
+* [Description](#description)
+* [Requirements](#requirements)
+* [How to use](#how-to-use)
+
```

Comment:
I would add also another point: "Expected results / output" - see the next comment.

Reply:
Done